src/pages/steps/AdditionalInformationStep.tsx

```diff
@@ -187,6 +192,30 @@ const AdditionalInformationStep = ({
     getEventByIdQuery.data,
   ]);
 
+  const priceInfo = useMemo(() => {
+    // @ts-expect-error
+    const priceInfo = getEventByIdQuery.data?.priceInfo ?? [];
+    // @ts-expect-error
+    const mainLanguage = getEventByIdQuery.data?.mainLanguage;
+
+    return priceInfo.map((rate: any) => {
```

Comment:
```suggestion
    return priceInfo.map((rate: Rate) => {
```

Is there no better typing that you already defined for `priceInfo` or `rate`?

Reply:
Here I cannot use it, since the rate we receive van the API is slightly different.
The `price` we receive from the api is a `number` vs in the formdata it's a `string`

Reply:
yes `Rate` is better here 👍